okta/resource_okta_app_group_assignments.go

```diff
@@ -17,7 +17,38 @@ func resourceAppGroupAssignments() *schema.Resource {
 		ReadContext:   resourceAppGroupAssignmentsRead,
 		DeleteContext: resourceAppGroupAssignmentsDelete,
 		UpdateContext: resourceAppGroupAssignmentsUpdate,
-		Importer:      &schema.ResourceImporter{},
+		Importer: &schema.ResourceImporter{
```

Comment:
I think it can be implemented even simpler:
```
Importer: &schema.ResourceImporter{
	StateContext: func(ctx context.Context, d *schema.ResourceData, m interface{}) ([]*schema.ResourceData, error) {
		_ = d.Set("app_id", d.Id())
		return []*schema.ResourceData{d}, nil
	},
},
```
The rest of the job will be done by `resourceAppGroupAssignmentsRead`

Reply:
@bogdanprodan-okta , 
I'm not a pro in Go or Terraform, so I'll take whatever you say :) So do you want me to resubmit the PR? Or will you do it yourself?

Reply:
Yeah, go ahead and change it. I'll approve and merge it asap.